arangod/Aql/MultiGet.h

```diff
@@ -0,0 +1,117 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2023 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Valery Mironov
+////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include "RocksDBEngine/RocksDBCollection.h"
+#include "RocksDBEngine/RocksDBFormat.h"
+#include "RocksDBEngine/RocksDBTransactionMethods.h"
+#include "RocksDBEngine/RocksDBColumnFamilyManager.h"
+
+#include <rocksdb/slice.h>
+#include <rocksdb/status.h>
+
+#include <string>
+#include <vector>
+
+namespace arangodb::aql {
+
+#if defined(ARANGODB_ENABLE_MAINTAINER_MODE) && defined(__linux__) && \
+    defined(__aarch64__)
+inline constexpr bool kEnableMultiGet = false;
```

Comment:
It looks strange that MultiGet is only disabled in maintainer mode on Linux on ARM, but not elsewhere.
There is probably a reason for this, but it should be mentioned here in a code comment.

Reply:
Forgot to comment this, idea is to test old behavior at least on some configuration

Reply:
Fixed

Reply:
Why don't we want to enable MultiGet globally?
Assuming that the MultiGet version works, is there a benefit in keeping the non-MultiGet version, but not making it available via configuration/opt-in somehow?

Reply:
Removed